scripts/ci/pre_commit/pre_commit_mypy.py

```diff
@@ -51,19 +55,26 @@
         "MOUNT_SOURCES": "selected",
     },
 )
+ci_environment = os.environ.get("CI")
 if res.returncode != 0:
+    if mypy_packages and ci_environment:
+        console.print(
+            "[yellow]You are running mypy with the packages selected. If you want to"
+            "reproduce it locally, you need to run the following command:\n"
+        )
+        console.print(
+            f'MYPY_PACKAGES="{mypy_packages}" pre-commit run --hook-stage manual mypy --all-files\n'
```

Comment:
Besides hiding the 4 options behind the ENV `MYPY_PACKAGES` have you also considered having 4 configs in pre-commit which you can directly run outside breeze? Might be a matter of taste though trading python scripting in breeze with 4-times a config in pre-commit.

It would then be
`pre-commit run --hook-stage manual mypy-airflow`
...instead of...
`MYPY_PACKAGES=airflow pre-commit run --hook-stage manual mypy --all-files`

Reply:
We could.. We will have the 8 pre-commit checks if we do it - 4 locally and for for CI... That's a lot :D

Reply:
I think one check with variable (especially that usually it will be run only when CI fails and you will get clear instructions) is probably better UX experience.

Reply:
Actually you were right @jscheffl  -> this way is better https://github.com/apache/airflow/pull/36760